infra/nnfw/cmake/packages/GLib2.0Config.cmake

```diff
@@ -0,0 +1,43 @@
+function(_GLIB_2_0_import)
+  find_library(GLIB_LIBRARIES
+    NAMES glib-2.0
+    ONLY_CMAKE_FIND_ROOT_PATH)
+  
+  get_filename_component(GLIB_LIBRARY_DIR ${GLIB_LIBRARIES} DIRECTORY)
+  find_path(GLIBCONFIG_INCLUDE_DIR
+    NAMES glibconfig.h
+    PATHS ${GLIB_LIBRARY_DIR}
+    PATH_SUFFIXES glib-2.0/include
+    CMAKE_FIND_ROOT_PATH_BOTH)
```

Comment:
Why this `find_path` use `CMAKE_FIND_ROOT_PATH_BOTH`?

Reply:
It considers the case where the `glibconfig.h` file is not located in `GLIB_LIBRARY_DIR`.

Reply:
My comment means why we need to remain possibility to search host by setting `CMAKE_FIND_ROOT_PATH_BOTH`.

Reply:
`glibconfig.h` is under `lib/` directory, so we pass `GLIB_LIBRARY_DIR` as hint. `GLIB_LIBRARY_DIR` is host path, so we need to find `glibconfig.h` based on host. 
It's better to use `NO_CMAKE_FIND_ROOT_PATH`, and add comment about this.

Reply:
Thank you. I applied your suggestion to the latest patch.